source/features/sticky-sidebar.css

```diff
@@ -22,3 +22,13 @@
 .rgh-sticky-sidebar + .border-top {
 	display: none;
 }
+
+/* Restores the top margin on repo root */
```

Comment:
What do you mean by restore? It's setting it to 0

Reply:
Because GitHub is setting it to negative margin. But you are right, it should probably be `reset` instead of `restore`.